README.md

```diff
@@ -273,6 +273,9 @@ pilot...
 # Need help?
 Please submit an issue on GitHub and provide information about your setup
 
+# Code of Conduct
+Help us keep pilot open and inclusive. Please read and follow our [Code of Conduct](https://github.com/fastlane/code-of-conduct).
```

Comment:
The name of the tool should be surrounded with `, so it looks like this `pilot`. How about we just generalise this to say

> Help us keep `fastlane` open and inclusive....